publick8s.tf

```diff
@@ -367,6 +367,48 @@ module "cronjob_geoip_data_fileshare_serviceprincipal_writer" {
   default_tags                   = local.default_tags
 }
 
+
+resource "azurerm_storage_share" "geoip_data_staging" {
+  name                 = "geoip-data-staging"
+  storage_account_name = azurerm_storage_account.publick8s.name
+  quota                = 1 # GeoIP databses weight around 80Mb
+}
+resource "kubernetes_namespace" "geoip_data_staging" {
+  provider = kubernetes.publick8s
+
+  metadata {
+    name = azurerm_storage_share.geoip_data_staging.name
+  }
+}
+resource "kubernetes_secret" "geoip_data_staging" {
+  provider = kubernetes.publick8s
+
+  metadata {
+    name      = azurerm_storage_share.geoip_data_staging.name
+    namespace = azurerm_storage_share.geoip_data_staging.name
+  }
+
+  data = {
+    azurestorageaccountname = azurerm_storage_account.publick8s.name
+    azurestorageaccountkey  = azurerm_storage_account.publick8s.primary_access_key
+  }
+
+  type = "Opaque"
+}
+
```

Comment:
```suggestion
```

=> uneeded as the secret is scoped to the storage account (e.g. it's the same for all file shares of a given storage account)

Reply:
see commit 16de42e